src/FakeItEasy.Analyzer/NonVirtualSetupAnalyzer.cs

```diff
@@ -85,24 +84,30 @@ private static bool IsSetupInvocation(SyntaxNodeAnalysisContext context, Invocat
 
         private static ImmutableDictionary<string, DiagnosticDescriptor> CreateDiagnosticsMap()
         {
-            var memberNames = DiagnosticSubjects.CallSpecMemberNames();
-
-            return memberNames.ToImmutableDictionary(name => name, name => DiagnosticDefinitions.NonVirtualSetup);
+            var callSpecMemberNames = new[]
+                        {
+                "FakeItEasy.A.CallTo",
+                "FakeItEasy.A.CallTo`1",
+                "FakeItEasy.A.CallToSet`1",
+                "FakeItEasy.Fake`1.CallsTo`1"
+            };
+
+            return callSpecMemberNames.ToImmutableDictionary(name => name, name => DiagnosticDefinitions.NonVirtualSetup);
         }
 
-        private static bool NotMethod(SymbolInfo x)
+        private static bool IsProperty(SymbolInfo symbolInfo)
         {
-            return x.Symbol?.Kind != SymbolKind.Method;
+            return symbolInfo.Symbol?.Kind == SymbolKind.Property;
         }
 
-        private static bool Method(SymbolInfo x)
+        private static bool IsMethod(SymbolInfo symbolInfo)
         {
-            return x.Symbol?.Kind == SymbolKind.Method;
+            return symbolInfo.Symbol?.Kind == SymbolKind.Method;
         }
 
-        private static bool Interface(SymbolInfo x)
+        private static bool IsInterface(SymbolInfo symbolInfo)

```

Comment:
Perhaps this should be renamed to `IsInterfaceMember`, since it doesn't actually check if `symbolInfo` is an interface, but rather if its containing type is an interface.